openedx/features/qverse_features/registration/tests/test_models.py

```diff
@@ -0,0 +1,65 @@
+"""
+Unit tests for QVerse registration app models.
+"""
+import mock
+import os
+
+from django.contrib.auth.models import User
+from django.core.files.base import ContentFile
+from django.test import TestCase
+
+from openedx.features.qverse_features.registration.models import BulkUserRegistration, QVerseUserProfile
+from openedx.features.qverse_features.registration.tests.factories import DepartmentFactory, QVerseUserProfileFactory
+from student.models import UserProfile
+
+
+@mock.patch(
```

Comment:
Just a little suggestion, instead of using class decorators use setUp/tearDown patcher pattern.

Reply:
Moved to function level as suggested instead of using the patcher pattern.